python-avd/pyavd/_eos_designs/structured_config/base/snmp_server.py

```diff
@@ -187,35 +198,84 @@ def _snmp_hosts(self: AvdStructuredConfigBaseProtocol, snmp_settings: EosDesigns
         self.structured_config.snmp_server.hosts = snmp_hosts
 
     def _snmp_local_interfaces(self: AvdStructuredConfigBaseProtocol, source_interfaces_inputs: EosDesigns.SourceInterfaces.Snmp) -> None:
-        """Set local_interfaces if "source_interfaces.snmp" is set."""
+        """
+        Set local_interfaces from "source_interfaces.snmp" or "snmp_settings.vrfs[].source_interface.
```

Comment:
Doh I had forgotten to cleanup from previous iterations.

```suggestion
        Set local_interfaces from "source_interfaces.snmp".
```